playbooks/fix-duplicate-hypervisor.yml

```diff
@@ -0,0 +1,28 @@
+---
+# Copyright 2017, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- name: Fix duplicate hypervisor
+  hosts: utility_all[0]
+  user: root
+  tasks:
+    - name: Rename node in nova intances table
+      shell: ". {{ ansible_env.HOME }}/openrc; mysql -BN -e \"update instances set node=concat(node, '.openstack.local') where node not like '%openstack.local'\" nova"
+      args:
+        executable: /bin/bash
+
+    - name: Delete old hypervisor
+      shell: ". {{ ansible_env.HOME }}/openrc; mysql -BN -e \"delete from compute_nodes where hypervisor_hostname not like '%openstack.local'\" nova "
```

Comment:
Don't think it's necessary as the old services are being cleaned via https://github.com/openstack/openstack-ansible-ops/blob/master/leap-upgrades/upgrade-utilities/post-redeploy-cleanup.yml

Reply:
OK, good to confirm no duplicated hypervisors existing but only updated hypervisor hostname.